src/components/ApplicationEnvironment/ApplicationEnvironmentSidePanel.tsx

```diff
@@ -0,0 +1,144 @@
+      </DrawerActions>
+      <div className="application-environment-details__component-side-panel__header">
+        <h1 className="application-environment-details__component-side-panel__header-heading">
+          <div className="application-environment-details__component-side-panel__name">{name}</div>
+          <div className="application-environment-details__component-side-panel__actions">
+            <SidePanelActionMenu component={component} />
+          </div>
+        </h1>
+      </div>
+      <DrawerPanelBody>
+        <DescriptionList
+          columnModifier={{
+            default: '1Col',
+          }}
+        >
+          {routesLoaded ? (
+            <>
+              {componentRouteWebURL && (
+                <DescriptionListGroup>
+                  <DescriptionListTerm>Route</DescriptionListTerm>
+                  <DescriptionListDescription>
+                    <ExternalLink href={componentRouteWebURL} text={componentRouteWebURL} />
+                  </DescriptionListDescription>
+                </DescriptionListGroup>
+              )}
+            </>
+          ) : (
+            <DescriptionListGroup>
+              <DescriptionListTerm>Route</DescriptionListTerm>
+              <DescriptionListDescription>
+                <Bullseye>
+                  <Spinner size="md" />
+                </Bullseye>
+              </DescriptionListDescription>
+            </DescriptionListGroup>
+          )}
+          <DescriptionListGroup>
+            <DescriptionListTerm>Revision</DescriptionListTerm>
+            <DescriptionListDescription>
+              <ApplicationEnvironmentRevisionLink component={component} />
+            </DescriptionListDescription>
+          </DescriptionListGroup>
+          {containerImage && (
+            <DescriptionListGroup>
+              <DescriptionListTerm>Image</DescriptionListTerm>
+              <DescriptionListDescription>
+                <ExternalLink href={`https://${containerImage}`} text={containerImage} />
+              </DescriptionListDescription>
+            </DescriptionListGroup>
+          )}
+          {resourceRequests && (
+            <DescriptionListGroup>
+              <Flex spaceItems={{ default: 'spaceItems4xl' }}>
+                <FlexItem>
+                  <DescriptionListTerm>CPU</DescriptionListTerm>
+                  <DescriptionListDescription>{resourceRequests.cpu}</DescriptionListDescription>
+                </FlexItem>
+                <FlexItem>
+                  <DescriptionListTerm>Memory</DescriptionListTerm>
+                  <DescriptionListDescription>{resourceRequests.memory}</DescriptionListDescription>
+                </FlexItem>
+              </Flex>
+            </DescriptionListGroup>
+          )}
+          <DescriptionListGroup>
+            <DescriptionListTerm>Instances</DescriptionListTerm>
+            <DescriptionListDescription>{replicas}</DescriptionListDescription>
+          </DescriptionListGroup>
```

Comment:
`replicas` is optional field.
```suggestion
  {replicas && (
          <DescriptionListGroup>
            <DescriptionListTerm>Instances</DescriptionListTerm>
            <DescriptionListDescription>{replicas}</DescriptionListDescription>
          </DescriptionListGroup>
         )}
```